pom.xml

```diff
@@ -14,7 +14,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <grpc.version>1.42.1</grpc.version>
+    <grpc.version>1.59.0</grpc.version>
```

Comment:
is there a need to change the gRPC version ? Can this be done as a separate PR to separate out the concerns from this one?

Reply:
From https://github.com/dapr/java-sdk/pull/947, I add `newUuid` which requires durabletask-client version 1.5.0. In v1.5.0 we also upgrade the grpc version to 1.59.0, if I don't upgrade grpc version here the examples break. So I have to include this update in this PR.